kernel/src/syscall/rmdir.rs

```diff
@@ -24,7 +24,10 @@ pub(super) fn sys_rmdirat(
 
     let (dir_dentry, name) = {
         let path_addr = path_addr.to_string_lossy();
-        if path_addr == "/" {
+        if path_addr.is_empty() {
```

Comment:
Could you do me a favor by renaming `path_addr` to `path`? It has been a very bad taste for naming.

```rust
let path = ctx.user_space().read_cstring(path_addr, MAX_FILENAME_LEN)?;
```

Reply:
Updated